apps/web/client/src/app/project/[id]/_components/editor-bar/inputs/color-picker.tsx

```
@@ -110,6 +117,291 @@ export const ColorPickerContent: React.FC<ColorPickerProps> = ({
+        try {
+            const normalized = cssValue.trim();
+
+            const linearMatch = normalized.match(/linear-gradient\(([^)]+)\)/);
+            const radialMatch = normalized.match(/radial-gradient\(([^)]+)\)/);
+            const conicMatch = normalized.match(/conic-gradient\(([^)]+)\)/);
+
+            let type: GradientState['type'] = 'linear';
+            let angle = 90;
+            let stopsString = '';
+
+            if (linearMatch && linearMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)